.github/workflows/standards-and-test.yml

```diff
@@ -45,33 +58,64 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
-          tools: phpcs, composer
+          tools: composer
           coverage: pcov
 
-      - name: Install PHP dependencies
-        run: |
-          export PATH="$HOME/.composer/vendor/bin:$PATH"
-          composer install --no-interaction
-          git clone --depth=1 https://github.com/pressbooks/pressbooks.git ../pressbooks
-          cd ../pressbooks && composer install --no-dev
-          cd ../pressbooks-plugin-scaffold
-
-      - name: Run PHP CodeSniffer
-        run: composer standards
+      - name: Install dependencies
+        env:
+          COMPOSER_AUTH: '{"github-oauth": {"github.com": "${{secrets.PAT_FOR_GITHUB_ACTIONS}}"} }'
+        run: composer install --no-interaction --no-progress
 
       - name: Install WP tests
-        run: bash bin/install-wp-tests.sh wordpress_test root root localhost ${{ matrix.wordpress }}
+        run: bash install-wp-tests.sh wordpress_test root root localhost ${{ matrix.wordpress }}
```

Comment:
@arzola removing `bin/` from the path here breaks this step of the CI pipeline because `install-wp-tests.sh` is still located in the `bin` folder in this repo.

Reply:
I changed the file